TODO

```diff
@@ -468,37 +468,6 @@ o SMP
   Priority:    Medium for SMP system.  Not critical to single CPU systems.
                NOTE:  There are no known bugs from this potential problem.
 
-  Title:       CORTEX-A GIC SGI INTERRUPT MASKING
```

Comment:
need update line 1 and 13 too

Reply:
> need update line 1 and 13 too

@xiaoxiang781216 
Thanks!
I forgot to update the lines in https://github.com/apache/incubator-nuttx/pull/2299
So I fixed the number in this PR (also added a comment)